src/pages/resultsView/expression/expressionHelpers.tsx

```diff
@@ -70,7 +70,7 @@ export const ExpressionStyleSheet: {
     fusion: {
```

Comment:
do we still have mutation type of `fusion`?

Reply:
Yes. My changes are just on study-view. other pages are yet to be done.